### ISSUE TITLE
JSON Http examples don't compile - code example all in one file

### ISSUE THREAD

**Issue by patientplatypus** [playframework/playframework#7705]
I've been working through your Scala JSONHTTP examples found here:

https://www.playframework.com/documentation/2.6.x/ScalaJsonHttp

And following the examples by copying and pasting in the right files leads to a huge amount of errors. Following the link at the bottom of the page led me to here: 

https://github.com/playframework/playframework/blob/2.6.x/documentation/manual/working/scalaGuide/main/json/code/ScalaJsonHttpSpec.scala

Which is just all the example snippets dumped in one file. As it stands right now I have no idea if the documentation is compileable or if there is an error on my machine. I understand that there is a compileable REST Api example, but I found that to have so many extra features that I found it very hard to follow. Please give a simple ~working~ example of writing one json and returning one json - the documentation is unclear.

**Comment by richdougherty** [comment]
Hi @patientplatypus, the `ScalaJsonHttpSpec.scala` is compiled as part of our continuous integration tests, so everything there should compile. To progress this issue we'll need more info from you to know what's going wrong. At least a snippet of the code you tried to compile and the compile errors you're getting. If you haven't verified that this is actually a Play bug then the mailing list is probably a better place to ask general usage questions: https://groups.google.com/forum/#!forum/play-framework. There are lots of people there who are happy to help.

> Please give a simple working example of writing one json and returning one json - the documentation is unclear.

If you've spotted a gap in the documentation it would be very helpful if you could submit something here - once you've got everything working/compiling of course.
